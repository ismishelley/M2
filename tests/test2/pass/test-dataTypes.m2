int main(){
	int a;
	float b;
	String c;
	bool d;

	a = 1;
	b = 2.0;
	c = "ah";
	d = true;

	printInt(a);
	printFloat(b);
	printStr(c);
	printBool(d);

	return 0;
}