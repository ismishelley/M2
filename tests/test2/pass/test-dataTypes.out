1	2.000000	ah
1	